yarn.lock

```diff
@@ -685,9 +685,9 @@
     to-fast-properties "^2.0.0"
 
 "@department-of-veterans-affairs/formation-react@^4.7.0":
-  version "4.7.0"
-  resolved "https://registry.yarnpkg.com/@department-of-veterans-affairs/formation-react/-/formation-react-4.7.0.tgz#669047353992e1803c096407e348940fe46d1ed8"
-  integrity sha512-ttCpkHaSjzshsfEBXiTMhWsVhpLnX6D6GB6x99owagAaJI2JIZy71QZbO4ZuEZvpAaVGmbqzSjIdM9HfGFBezg==
+  version "4.8.0"
```

Comment:
This is to pull in the change where I can set the `level` prop in the alert box and control which `h` element is used.

Reply:
I'm not seeing the same change to `package.json`?

Reply:
The version is within the range specified in package.json, so yarn didn't change it. Yarn won't update it until we get past 5.0.

Reply:
Oh right, I still just have the habit of explicitly updating `package.json` in cases like this but I know that isn't standard practice.

Just curious: what `yarn` command do you run to fetch the latest and greatest package from NPM that meets what we specify in `package.json`?

Reply:
I use `yarn upgrade <package name>`, and then when I need to get the absolute most recent, `yarn upgrade <package name> --latest`

Reply:
Cool, that's more efficient than blowing away `node_modules` and rerunning `yarn` :-p